spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java

```diff
@@ -314,10 +310,31 @@ public DeviceResult run(AndroidDebugBridge adb) {
     return result.build();
   }
 
+  private void addCodeCoverageInstrumentationArgs(RemoteAndroidTestRunner runner, IDevice device)
```

Comment:
Code style checker complained that `run` method got too long, so I created couple of methods.